forc-plugins/forc-crypto/src/keygen/new_key.rs

```diff
@@ -0,0 +1,60 @@
+//! This file will be hosted here until
+//! https://github.com/FuelLabs/sway/issues/5170 is fixed
+use super::KeyType;
+use anyhow::Result;
+use fuel_core_types::{
+    fuel_crypto::{
+        rand::{prelude::StdRng, SeedableRng},
+        SecretKey,
+    },
+    fuel_tx::Input,
+};
+use libp2p_identity::{secp256k1, Keypair, PeerId};
+use serde_json::json;
+use std::ops::Deref;
+
+/// Generate a random new secret & public key in the format expected by fuel-core
+#[derive(Debug, clap::Args)]
+#[clap(author, version, about = "Creates a new key for use with fuel-core")]
+pub struct Arg {
+    /// Key type to generate. It can either be `block-production` or `p2p`.
```

Comment:
This is what the help message looks like: 
<img width="874" alt="image" src="https://github.com/FuelLabs/sway/assets/47993817/4d33a27e-858a-4b3f-9afb-ee9a4b264595">


It says that `p2p` is a possible value, which doesn't seem to work: 
<img width="560" alt="image" src="https://github.com/FuelLabs/sway/assets/47993817/6f827f8d-9495-45e4-acf8-af5e479a2783">

`peering` does work as a value: `forc crypto new-key -k peering`